projects/demo-integrations/cypress/plugins/index.ts

```diff
@@ -1,8 +1,14 @@
-const {addMatchImageSnapshotPlugin} = require('cypress-image-snapshot/plugin');
-const {viewportWidth, viewportHeight} = require('./../../cypress.json');
+// dont change import to `@taiga-ui/testing/cypress` until merging of this PR
+// https://github.com/jaredpalmer/cypress-image-snapshot/pull/250
+import {tuiAddSnapshotPlugin} from '@taiga-ui/testing/cypress/snapshot/plugin';
 
-module.exports = (on, config) => {
-    addMatchImageSnapshotPlugin(on, config);
+import {viewportHeight, viewportWidth} from './../../cypress.json';
+
+export default (on: Cypress.PluginEvents, config: Cypress.PluginConfigOptions): void => {
+    tuiAddSnapshotPlugin(on, config, {
+        newSnapshotMark: '==[new]==',
```

Comment:
what do you think about use predicate transformer instead of ..?

```ts
tuiAddSnapshotPlugin(on, config, {
        newSnapshotMark: (name) => `==[new]==${name}`, // Users control how they name files
});
```

Reply:
✅